book-examples/leptos/src/default.rs

```diff
@@ -3,6 +3,9 @@ mod components;
 #[cfg(feature = "button")]
 mod button;
 
+#[cfg(feature = "card")]
+mod card;
```

Comment:
If you remove the blank line between such `mod` statements, Cargo will sort them alphabetically.